tests/resources/test-types.json

```
@@ -81,15 +83,15 @@
           {
             "forVehicleType": "psv",
             "forVehicleSize": "large",
-            "forVehicleConfiguration": "rigid",
+            "forVehicleConfiguration": "rigid/articulated",
```

As discussed, to make the app filter correctly these test-types, can you please change these occurrences with an array:
```suggestion
            "forVehicleConfiguration": [
              "articulated",
              "rigid"
            ],
```

---

if the search is for the exact value i.e. rigid or articulated then "rigid/articulated" would also work. `forVehicleConfiguration` for all `testCodes` arrays is string. If some of them are modified to an array then it would create confusion.

---

You are right, the filtering does work. 
Since this test-type configuration is used also by the test-result service (to add the correct testCode to the test-result), I tried to submit a test with `rigid/articulated` and the submission failed. I think you might need to change it

---

fixed as suggested.